Website/htdocs/mpmanager/resources/assets/js/components/MiniGrid/Dashboard.vue

```diff
@@ -622,6 +642,17 @@ export default {
         }
     },
     methods: {
+        setMiniGrid(miniGridId){
+            this.$router.replace('/dashboards/mini-grid/' + miniGridId)
+        },
+        async getMiniGridList() {
+            try {
+                this.miniGrids = await this.miniGridService.getMiniGrids()
+            } catch (e) {
+                this.alertNotify('error', e.message)
```

Comment:
could you replace that with "Error: failed to get MiniGrid list". It is not good to show some server-side errors to the client